koku/reporting/provider/gcp/models.py

```diff
@@ -785,3 +787,71 @@ class Meta:
     account_id = models.CharField(max_length=50, null=False)
 
     region = models.CharField(max_length=50, null=True)
+
+
+class GCPNetworkSummary(models.Model):
+    """A MATERIALIZED VIEW specifically for UI API queries.
+
+    This table gives a daily breakdown of network usage.
+
+    """
+
+    class Meta:
+        """Meta for GCPNetworkSummary."""
+
+        db_table = "reporting_gcp_network_summary"
+        managed = False
+
+    id = models.IntegerField(primary_key=True)
+
+    usage_start = models.DateField(null=False)
+
+    usage_end = models.DateField(null=False)
+
+    account_id = models.CharField(max_length=50, null=False)
+
+    usage_amount = models.DecimalField(max_digits=24, decimal_places=9, null=True)
```

Comment:
I need to update the usage amount in these to match: https://github.com/project-koku/koku/pull/2648

Reply:
I lied, we use the `usage_in_pricing_units` here.